runtime/assembly/api/tests/storage.ts

```diff
@@ -25,8 +25,47 @@ export function test_storage_set(data: i32, len: i32): i64 {
  * @param data - i32 pointer to the start of the arguments passed
  * @param len - i32 length (in bytes) of the arguments passed
  */
-export function test_storage_get(data: i32, len: i32): i64 {
+export function test_storage_get(data: i32, len: i32): u64 {
     let input = Serialiser.deserialiseInput(data, len);
     const value = Storage.get(input);
     return value.isSome() ? Serialiser.serialiseResult((<ByteArray>value.unwrap()).values) : Serialiser.serialiseResult([]);
-}
\ No newline at end of file
+}
+
+/**
+ * Test read method of storage
+ * @param data - i32 pointer to the start of the arguments passed
+ * @param len - i32 length (in bytes) of the arguments passed
+ */
+export function test_storage_read(data: i32, len: i32): u64 {
```

Comment:
This function will be slightly refactored once we put the buffer outside the `Storage.read`